src/ecmult_impl.h

```diff
@@ -190,6 +190,23 @@ static void secp256k1_ecmult_odd_multiples_table_storage_var(const int n, secp25
     secp256k1_fe_sqr(&dx_over_dz_squared, &d.z);
     secp256k1_fe_mul(&dx_over_dz_squared, &dx_over_dz_squared, &d.x);
 
+    /* Going into the second loop, we have set `pre[n-1]` to its final affine
+     * form, but still need to set `pre[i]` for `i` in 0 through `n-2`. We
+     * we have `zi = (p.z * d.z)^-1`, where
+     *
+     *     `p.z` is the z-coordinate of the point which was assigned to
+     *           `pre[n-1]`; by multiplying this inverse by successive
+     *           z-ratios we will obtain the inverse z-coordinate of all
+     *           other points. These inverses would have been expensive
+     *           to compute directly.
+     *     `d.z` is the multiplier that must be applied to all z-coordinates
+     *           to move from our isomorphic curve back to secp256k1. We
+     *           applied this multiplier above to the inverse-z-coordinate
+     *           computing `pre[n-1]`, and since all subsequent inverse-zs
+     *           are computed as multiples of this value, they will also
+     *           have the multiplier applied to them without any further
+     *           computation.
```

Comment:
Actually this block is confusing and does not distinguish between the two curves. Will rewrite.